src/Sylius/Bundle/AdminBundle/Controller/NotificationController.php

```diff
@@ -23,7 +23,7 @@
 /**
  * @author Jan Góralski <jan.goralski@lakion.com>
  */
-class NotificationController
+final class NotificationController
```

Comment:
It's a minor BC break so it's OK for it to be introduced but we'd still need some info in UPGRADE.md